eclipse/src/saros/editor/EditorManager.java

```diff
@@ -1725,4 +1730,9 @@ public boolean evaluate(Annotation annotation) {
   private void checkThreadAccess() {
     if (!SWTUtils.isSWT()) throw new IllegalStateException("method must be invoked from EDT");
   }
+
+  @Deprecated
```

Comment:
Why is this method deprecated? What should be used instead?

Reply:
It is only used by one disabled feature. I really do not know if it was a good design decision to allow every component to hook into the activity generation of every other component.

Reply:
Added it to the public API. However it is still not used.

Reply:
That is not what I meant. If you think the method should no longer be used, you can add it as deprecated. But, in that case, it should have a description of why it was marked as deprecated and what to do instead (i.e. which other method to use or what to do to replicate the functionality).